File: backend/src/main/java/top/vuhe/portal/service/intf/StatusService.java
package top.vuhe.portal.service.intf;

import com.baomidou.mybatisplus.extension.service.IService;
import top.vuhe.entity.equipment.dao.StatusDAO;

import java.util.Map;

/**
 * @author zhuhe
 */
public interface StatusService extends IService<StatusDAO> {
    /**
     * 获取状态值对应信息
     *
     * @return 状态信息
     */
    Map<Integer, StatusDAO> getStatusInfo();
}

File: backend-kt/src/main/kotlin/top/vuhe/sw/portal/service/StatusService.kt
package top.vuhe.sw.portal.service

import com.baomidou.mybatisplus.extension.service.IService
import top.vuhe.sw.entity.equipment.dao.StatusDAO

interface StatusService: IService<StatusDAO> {
    /**
     * 获取状态值对应信息
     *
     * @return 状态信息
     */
    fun getStatusInfo(): Map<Int?, StatusDAO>
}